src/classes/BDI_DataImport_TEST.cls

```diff
@@ -1501,15 +1619,15 @@ public with sharing class BDI_DataImport_TEST {
     **********************************************************************************************************/
     private static List<DataImport__c> getDIs() {
```

Comment:
Just stylistic, but we might consider changing this to getDataImports() ?

Reply:
But if you give a mouse a cookie, the mouse will have to update all the listDI references too 😆 fair suggestion though, and the diff is quite a bit larger now!